corehq/apps/domain/calculations.py

```diff
@@ -33,24 +38,39 @@ def num_mobile_users(domain, *args):
 
 def active_mobile_users(domain, *args):
     """
-    Returns the number of mobile users who have submitted a form in the last 30 days
+    Returns the number of mobile users who have submitted a form or SMS in the
+    last 30 days
     """
     now = datetime.now()
-    then = (now - timedelta(days=30)).strftime(DATE_FORMAT)
-    now = now.strftime(DATE_FORMAT)
+    then = (now - timedelta(days=365))
+
+    user_ids = get_mobile_users(domain)
+
+    form_users = {q['term'] for q in (
+        FormES()
+        .domain(domain)
+        .user_facet(size=USER_COUNT_UPPER_BOUND)
+        .submitted(gte=then)
+        .user_id(user_ids)
+        .size(0)
+        .run()
+        .facets.user.result
+    )}
+
+    sms_users = {q['term'] for q in (
+        SMSES()
+        .user_facet(size=USER_COUNT_UPPER_BOUND)
+        .to_commcare_user()
+        .domain(domain)
+        .received(gte=then)
+        .size(0)
+        .run()
+        .facets.user.result
+    )}
+
+    num_users = len(form_users | sms_users)
+    return num_users if 'inactive' not in args else len(user_ids) - num_users
```

Comment:
This never actually worked in the previous implementation.

Active and inactive returned the same thing.